_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+|:----------:|:----:|:---------:|:------------:|:-------:|:-----:|:----:|:------------:|
+| 1.31-1.33  | 0.26 | 5.4.3     | 1.16.1       | 1.16    | 1.24  | 2.41 | 7.7          |
+
+### Security:
+
+* CVE reductions - regular scanning with trivy
+* Kubernetes and container security best practices:
+    * [Rootless containers](https://github.com/kubeflow/manifests/issues/2528) / PodSecurityStandards restricted for:
+      Istio-CNI, Knative, Dex, Oauth2-proxy, Spark
+    * [50 % done](https://github.com/kubeflow/manifests/pull/3050): KFP, Notebooks / Workspaces, Katib, Trainer, Kserve, …
+    * Istio-CNI as default for rootless Kubeflow postponed to [1.10.1](https://github.com/kubeflow/manifests/milestone/2)
+* OIDC-authservice has been replaced by oauth2-proxy
+* [Oauth2-proxy](https://github.com/kubeflow/manifests#oauth2-proxy) and [Dex](https://github.com/kubeflow/manifests#dex)
+  documentation for external OIDC authentication (Keycloak, and OIDC providers such as Azure, Google etc.)
+
+Trivy CVE scans March 25 2025:
+
+|     Working Group      | Images | Critical CVE | High CVE | Medium CVE | Low CVE |
+|:----------------------:|:------:|:------------:|:--------:|:----------:|:-------:|
+|         Katib          |   17   |      11      |   101    |    417     |   734   |
+|       Pipelines        |   15   |      57      |   490    |    4030    |   1922  |
+| Workbenches(Notebooks) |   12   |      12      |    59    |    179     |   224   |
+|         Kserve         |   16   |      21      |   305    |    6803    |   1588  |
+|       Manifests        |   14   |      8       |    4     |     94     |    52   |
+|        Trainer         |   1    |      0       |    0     |     1      |    0    |
+|     Model Registry     |   6    |      1       |    13    |    153     |   188   |
+|         Spark          |   1    |      5       |    37    |    1640    |   141   |
+|       All Images       |   81   |     115      |   1009   |   13275    |   4804  |
+
+## Pipelines
+
+Kubeflow Pipelines 2.4.1 introduces several enhancements to improve the flexibility and control of your machine learning
+workflows.
+
+### Support for Placeholders in Resource Limits
+
+Kubeflow Pipelines 2.4.1 introduces support for [placeholders in resource limits](https://github.com/kubeflow/pipelines/pull/11501),
+enhancing flexibility in pipeline execution.This update allows users to define dynamic resource limits using
+parameterized values, enabling more adaptable and reusable pipeline definitions.
+
+### Support for Loop Parallelism
+
+Kubeflow Pipelines 2.4.1 introduces a new [Parallelism Limit for `ParallelFor` tasks](https://github.com/kubeflow/pipelines/issues/8718),
+giving users the ability to run massively parallel inference pipelines, with more control over parallel execution in
+their workflows. This feature allows users to specify the maximum number of parallel iterations, preventing resource
+overutilization and improving system stability.
+
```

Comment:
again, potentially quantify.     When running large pipelines with GPUs, proper use of this feature could save your team thousands of dollars in compute expenses.